testgrid/tgrun/pkg/scheduler/addons.go

```diff
@@ -34,6 +34,9 @@ func generateAllAddOns() ([]kurlv1beta1.InstallerSpec, error) {
 	return installerSpecs, nil
 }
 
+// hack
+var previouslyGeneratedNames = []string{}
```

Comment:
what's this used for? (I realize it's not new)

Reply:
i have no idea. i moved it closer to where its referenced